packages/monitoring-system/data-processor/cloudbuild.yaml

```diff
@@ -0,0 +1,43 @@
+# Copyright 2020 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+
+steps:
+  - name: gcr.io/cloud-builders/npm
+    id: "compile"
+    waitFor: ["-"]
+    dir: './packages/monitoring-system/data-processor'
+    entrypoint: npm
+    args: ['run', 'clean-compile']
+
+  - name: gcr.io/cloud-builders/docker
+    id: "build-docker"
+    waitFor: ["compile"]
+    dir: './packages/monitoring-system/data-processor'
+    entrypoint: docker
+    args: ['build', '--tag', 'data-processor:1.0', '.']
```

Comment:
Is  the idea that this would just push over top of the same tag on each deploy? I think this could potentially have some unexpected side effects, as I'd be surprised if containers aren't being cached.

Reply:
I've been overwriting the same container tag so far for all deployments and haven't really run into an issue, but it might be different for Cloud Build.

There's not really any reason to create a new tag for each release because I don't expect we'll reuse old images again.